pkg/block/block_test.go

```diff
@@ -87,12 +87,12 @@ func TestUpload(t *testing.T) {
 
 	bkt := objstore.NewInMemBucket()
 	b1, err := e2eutil.CreateBlock(ctx, tmpDir, []labels.Labels{
-		{{Name: "a", Value: "1"}},
-		{{Name: "a", Value: "2"}},
-		{{Name: "a", Value: "3"}},
-		{{Name: "a", Value: "4"}},
-		{{Name: "b", Value: "1"}},
-	}, 100, 0, 1000, labels.Labels{{Name: "ext1", Value: "val1"}}, 124, metadata.NoneFunc)
+		labels.New(labels.Label{Name: "a", Value: "1"}),
```

Comment:
Wouldn't this error with stringlabels enabled?

Reply:
I think we can just use `labels.FromStrings` here

Reply:
We can but it wont error with stringlabels ( i tried actually ); so i didnt bother since its many lines to edit.